src/XMakeBuildEngine/Construction/Solution/ProjectInSolution.cs

```diff
@@ -305,11 +305,8 @@ internal bool CanBeMSBuildProjectFile(out string errorMessage)
 
                 if (mainProjectElement != null && mainProjectElement.LocalName == "Project")
                 {
-                    if (String.Compare(mainProjectElement.NamespaceURI, XMakeAttributes.defaultXmlNamespace, StringComparison.OrdinalIgnoreCase) == 0)
-                    {
-                        _canBeMSBuildProjectFile = true;
-                        return _canBeMSBuildProjectFile;
-                    }
+                    _canBeMSBuildProjectFile = true;
```

Comment:
Ah, that works. I was thinking that you removed the only check--but it's still inside the if-<Project> one.